### PR TITLE
update gcloud SDK to 202.0.0

### PR DESCRIPTION
**Pull request by montmanu**
See [release notes](https://cloud.google.com/sdk/docs/release-notes#20200_2018-05-22)

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -2,7 +2,7 @@ FROM alpine:3.4
 
 RUN apk add --no-cache curl python
 
-ENV GOOGLE_CLOUD_SDK_VERSION=187.0.0
+ENV GOOGLE_CLOUD_SDK_VERSION=202.0.0
```
Comment (yunzhu-li) [inline Dockerfile]: The latest version is `207.0.0`, is there a reason not to update to the latest?

**Comment by tonglil** [comment]
NOTE: this is to update kubectl from 1.8.6 to 1.9.7.

File: Dockerfile (hunk 1 above)
Comment (montmanu) [reply Dockerfile]: not necessarily, would that be preferred?

**Comment by tonglil** [comment]
@yunzhu-li at one point I want to create releases that disconnect drone-gke from kubectl, similar to [drone-terraform](https://hub.docker.com/r/jmccann/drone-terraform/tags/), by using multiple Dockerfiles:
- `Dockerfile-1.8` -> uses the SDK base image that contains kubectl 1.8.*
- `Dockerfile-1.9` -> uses the SDK base image that contains kubectl 1.9.*
- `Dockerfile` -> uses the SDK:latest base image

Any minor version updates can just PR against the appropriate Dockerfile.

File: Dockerfile (hunk 1 above)
Comment (montmanu) [reply Dockerfile]: see https://github.com/NYTimes/drone-gke/pull/89

**Pull request closed without merging**